include/picongpu/plugins/openPMD/openPMDWriter.def

```diff
@@ -97,8 +91,6 @@ namespace picongpu
             std::unique_ptr<AbstractJsonMatcher> jsonMatcher;
             std::string jsonRestartParams;
```

Comment:
IMO this is not required anymore an could lead to the wrong usage of this variable instead of the value from the ThreadParameters.

Reply:
Agreed, I will push a commit that merges `PluginParameters` and `ThreadParams`